dashboard/lib/services/curriculum_pdfs.rb

```diff
@@ -128,8 +129,24 @@ def self.regenerate_pdfs(scripts)
 
           script.lessons.select(&:has_lesson_plan).each do |lesson|
             puts "Regenerating Lesson PDFs for #{lesson.key} (from #{script.name})"
-            generate_lesson_pdf(lesson, dir)
+            # generate PDF and get the pathname to the generated PDF
+            pdf_pathname = generate_lesson_pdf(lesson, dir)
             any_pdf_generated = true
+
+            # Generate matadata
+            if any_pdf_generated
+              metadata ={
+                course: script.name,
+                unit_fullname: script.localized_title,
```

Comment:
This way is consistent with the Google Drive export metadata.

```suggestion
                unit_fullname: script.name,
```

Reply:
Just to confirm - `script.name` would give you something like `csd3-2024` whereas `script.localized_title` gives you `‘Unit 3 – Interactive Animations and Games (‘23-’24)’`. I was basing this choice to use `localized title` after what was in [this doc](https://docs.google.com/document/d/1tgX8tSOVNIQ4v5CTkkA9xzsLzfIJKz9x1BVfN7CFpec/edit?tab=t.0).  Fine to change it to just `script.name` but wanted to confirm this first and then update the doc.

Reply:
Hmm...the localized title does look better and is more in line with the requirements from the spec. Let's standardize on that. Not 100% sure why we even need this field anyway.

Reply:
You know, as I think about it, the more readable format does look nice, BUT, I think it is possibly more prone to errors.  That is, if we change how we display the localized title, some of the PDFs will have incorrect metadata.  This makes me think that `script.name` is the right way to go to keep the metadata cleaner.